packages/orbit-components/src/Layout/LayoutColumn/index.jsx

```diff
@@ -28,9 +33,15 @@ StyledColumn.defaultProps = {
   theme: defaultTheme,
 };
 
-const LayoutColumn = ({ children, hideOn, as = "div", dataTest }: Props): React.Node => {
+const LayoutColumn = ({
+  children,
+  hideOn,
+  as = "div",
+  spanEntireRow = false,
```

Comment:
I do not think you should have it by default as `false`, because `undefiend` is also a falsy value